### PR TITLE
Consolidated QAT Standards 

### PR DESCRIPTION
**Pull request by gurukiranHMO**
We have had old PRs that were on various branch from QAT which were forked and added here
This is one PR consolidating all the PRs and making sure the new sign in process is followed and easier to address comments and merge here

Is this pull request a content or a code change? (Please fill in the relevant section and delete the other)

Content chang e

# Code change
I can confirm:
## Accessibility considerations
- [ ] Please review the [accessibility checks for layout changes](https://github.com/UKHomeOffice/engineering-guidance-and-standards/blob/main/technical-docs/accessibility/layout-checks.md).

- [x] This change will not change layouts, page structures or anything else that might impact accessibility

or
- [ ] This change might impact accessibility, automated aXe tests cover the impact

or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)

## Commit signing

- [x] All commits are signed with a key that can be verified by GitHub. [GitHub guidance on signing commits](https://docs.github.com/en/enterprise-cloud@latest/authentication/managing-commit-signature-verification/signing-commits)

# Content change 
- [ x] Please review the [accessibility checks for content changes](https://github.com/UKHomeOffice/engineering-guidance-and-standards/blob/main/technical-docs/accessibility/content-checks.md).

I can confirm:
- [ x] Content does not include any code or configuration changes (excluding frontmatter information)
- [x ] Content meets the content standards
e.g. [Writing a principle](https://engineering.homeoffice.gov.uk/standards/writing-a-principle/) and [Writing a standard](https://engineering.homeoffice.gov.uk/standards/writing-a-standard/)
- [ x] Content is suitable to open source, i.e.:
    - Content does not relate to unreleased gov policy
    - Content does not refer to anti-fraud mechanisms
    - Content does not include sensitive business logic
- [x ] Last updated date for content is correct
- [x ] All commits are signed with a key that can be verified by GitHub. [GitHub guidance on signing commits](https://docs.github.com/en/enterprise-cloud@latest/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

File: docs/standards/automated-testing-ci-cd-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Requirements
+
+- [You MUST automate code integration and building](#you-must-automate-code-integration-and-building)
+- [You MUST automate testing at multiple levels](#you-must-automate-testing-at-multiple-levels)
+- [You MUST use product pipeline to run tests](#you-must-use-product-pipeline-to-run-tests)
+- [You MUST automate feedback and reporting](#you-must-automate-feedback-and-reporting)
+- [You MUST advocate continuous monitoring and logging](#you-must-advocate-continuous-monitoring-and-logging)
+
+
+### 	You MUST automate code integration and building
```
Comment (edhamiltonHO) [inline docs/standards/automated-testing-ci-cd-pipeline.md:25]: While valid, I don't think this requirement belongs in this standard

File: docs/standards/automated-testing-ci-cd-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### 	You MUST advocate continuous monitoring and logging
+Implementing continuous monitoring of application performance and infrastructure health across all environments, from development to production, involves deploying automated tools and systems to track key metrics such as response times, error rates, resource utilisation, and security vulnerabilities. This proactive approach enables real-time visibility into system behaviour, allowing teams to identify and address potential issues before they impact end-users. Detailed dashboards and alerts provide immediate notifications of anomalies, facilitating rapid diagnosis and resolution. Benefits include improved application stability and reliability, reduced downtime, and enhanced user experience.
```
Comment (edhamiltonHO) [inline docs/standards/automated-testing-ci-cd-pipeline.md:39]: Similar to point on automated build above, this is covered elsewhere in SEGAS and not a requirement relating to automated test

File: docs/standards/test-coverage.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- [You MUST prioritise testing coverage on the expected behaviour of the code, including edge cases and boundary conditions](#you-must-prioritise-testing-coverage-on-the-expected-behaviour-of-the-code-including-edge-cases-and-boundary-conditions)
+- [You MUST prioritise testing critical or high-risk areas of the application](#you-must-prioritise-testing-critical-or-high-risk-areas-of-the-application)
+- [You MUST prioritise achieving coverage target for new code changes, legacy system unit tests should be secondary](#you-must-prioritise-achieving-coverage-target-for-new-code-changes-legacy-system-unit-tests-should-be-secondary)
+- [You MUST implement code coverage tools integrated into CI/CD pipelines for continuous measurement](#you-must-implement-code-coverage-tools-integrated-into-cicd-pipelines-for-continuous-measurement)
+- [You MUST have test coverage metrics emphasise a holistic approach to quality engineering](#you-must-have-test-coverage-metrics-emphasise-a-holistic-approach-to-quality-engineering)
+
+### You MUST have a minimum of 80% code coverage is required
```
Comment (edhamiltonHO) [inline docs/standards/test-coverage.md:32]: Need to rename this requirement for clarity

File: docs/standards/test-coverage.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Maintaining a minimum of 80% code coverage ensures that a significant portion of the codebase is tested, reducing the risk of undetected defects. This threshold helps improve software reliability, maintainability, and overall quality by verifying that critical logic paths, functions, and features are covered by automated tests. while 100% coverage is not always practical, setting a standard of 80% encourages consistency testing practices without excessive overhead.
+
+### You MUST implement static code vulnerability scans to identify security issues within the code
```
Comment (edhamiltonHO) [inline docs/standards/test-coverage.md:36]: While valid, this requirement does not relate to test coverage, so should be removed and be part of a different standard
